lightautoml/transformers/datetime.py

```diff
@@ -243,6 +259,10 @@ def fit(self, dataset: LAMLDataset) -> "LAMLTransformer":
                 self._features.append("season_{0}__{1}".format(s, col))
             if roles[col].country is not None:
                 self._features.append("season_hol__{0}".format(col))
+            if roles[col].last_work_day is not None:
```

Comment:
You didn't commit changes in datetime role and didn't add this argument, so there will be an error with argument `last_work_day`.